### DISCUSSION TITLE
Where to put `fastlane_session` parameter?

### DISCUSSION THREAD

**Discussion by NikKovIos** [fastlane/fastlane#17756] [General] [answered]
I have make a [question](https://stackoverflow.com/questions/65196972/where-to-put-fastlane-session-value) in Stack Overflow. Will duplicate it here.

I'm using fastlane with Fastfile and Appfile
According [this](https://github.com/fastlane/fastlane/tree/master/spaceship#2-step-verification) doc, i did create FASTLANE_SESSION variable, but can't figure out where and how to put it to make it work. I don't use CI service, only fastlane in command line to deploy my ios build. Help please.

**Comment by rogerluan** [comment, marked as answer]
Looks like your question has been answered on Stack Overflow already. Both [this](https://stackoverflow.com/a/65245876/4075379) and [this](https://stackoverflow.com/a/65208046/4075379) answers are correct and I also recommend using an API key instead.

Good luck!
